astrodendro/dendrogram.py

```diff
@@ -463,6 +468,23 @@ def __eq__(self, other):
         if not (self.data == other.data).all():
             return False
 
+        if self.params['min_value'] != other.params['min_value']:
+            return False
+
+        self_params = self.params
+        other_params = other.params
+
+        self_params.pop('min_value')
```

Comment:
ooo careful here -- `pop` permanently mutates the params dict of the parent dendrograms. Better to set `self_params = self.params.copy(), other_params = other.params.copy()`